File: soal6.ts
function birthdayCakeCandles(candles: number[]): number {
    const tallest = Math.max(...candles); // Cari tinggi lilin tertinggi
    return candles.filter(candle => candle === tallest).length; // Hitung jumlah lilin tertinggi
}

// Contoh penggunaan
const number = parseInt(prompt("Masukkan jumlah lilin: ") || "0");
const candles: number[] = prompt("Masukkan tinggi lilin (dipisah spasi): ")?.split(" ").map(Number) || [];

console.log(birthdayCakeCandles(candles));

File: soal6.py
def birthdayCakeCandles(candles):
    tallest = max(candles)  # Cari tinggi lilin tertinggi
    return candles.count(tallest)  # Hitung jumlah lilin dengan tinggi tersebut

# Contoh penggunaan
if __name__ == "__main__":
    n = int(input("Masukkan jumlah lilin: ").strip())
    candles = list(map(int, input().split()))
    print(birthdayCakeCandles(candles))
